test/browser/amplitudejs.html

```diff
@@ -27,7 +27,7 @@
                    'setOptOut', 'setVersionName', 'setDomain', 'setDeviceId',
                    'setGlobalUserProperties', 'identify', 'clearUserProperties',
                    'setGroup', 'logRevenueV2', 'regenerateDeviceId',
-                   'logEventWithTimestamp', 'logEventWithGroups'];
+                   'logEventWithTimestamp', 'logEventWithGroups', 'onInit'];
```

Comment:
ohh, this snippet is probably out of date. Thanks for adding this

Reply:
np 👍